### PR TITLE
Add session_hash

### PR DESCRIPTION
**Pull request by haileys**
cc @Sirupsen

### CODE REVIEW

File: db/migrate/20131021020547_add_session_hash_to_user_model.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class AddSessionHashToUserModel < ActiveRecord::Migration
+  def change
+    add_column :users, :session_hash, :string
+
+    execute "UPDATE users SET session_hash = '123deadbeef'"
```
Comment (sirupsen) [inline db/migrate/20131021020547_add_session_hash_to_user_model.rb:5]: app config?

Comment (haileys) [inline db/migrate/20131021020547_add_session_hash_to_user_model.rb:5]: nah this is just to satisfy the not null constraint

Comment (sirupsen) [inline db/migrate/20131021020547_add_session_hash_to_user_model.rb:5]: ah ok

Comment (haileys) [inline db/migrate/20131021020547_add_session_hash_to_user_model.rb:5]: There's no security issue here - the session_hash just needs to be set to _something_

**Comment by sirupsen** [comment]
:ship: :it:

**Pull request merged**